### PR TITLE
Implemented the exit code in the tools scripts

### PR DESCRIPTION
**Pull request by cmadjar**
### Added the exit codes to the following scripts in tools:
- BackPopulateSNRAndAcquisitionOrder.pl
- ProdToConfig.pl
- example_scripts/deletemincsqlwrapper.pl

### Notes: 
- no exits are present in seriesuid2fileid

Link to redmine ticket: https://redmine.cbrain.mcgill.ca/issues/14055

### CODE REVIEW

File: tools/BackPopulateSNRAndAcquisitionOrder.pl [hunk 1]
```diff
@@ -52,7 +54,7 @@
 if ($profile && !@Settings::db) {
     print "\n\tERROR: You don't have a configuration file named ".
           "'$profile' in:  $ENV{LORIS_CONFIG}/.loris_mri/ \n\n";
-    exit 2;
+    exit $NeuroDB::ExitCodes::DB_SETTINGS_FAILURE;
```
Comment (MounaSafiHarab) [inline tools/BackPopulateSNRAndAcquisitionOrder.pl]: @cmadjar 
did you not decide to split this condition into 2 in your other scripts?
you have done this in all otgher sctipts I think:

```
if ( !$profile ) {
    print $Help;
    print STDERR "$Usage\n\tERROR: missing -profile argument\n\n";
    exit $NeuroDB::ExitCodes::PROFILE_FAILURE;
}
{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
if ( !@Settings::db ) {
    print STDERR "\n\tERROR: You don't have a \@db setting in the file "
                 . "$ENV{LORIS_CONFIG}/.loris_mri/$profile \n\n";
    exit $NeuroDB::ExitCodes::DB_SETTINGS_FAILURE;
}
```

File: tools/ProdToConfig.pl [hunk 2]
```diff
@@ -33,24 +34,26 @@
...
 USAGE
 &Getopt::Tabular::SetHelp( $Help, $Usage );
-&Getopt::Tabular::GetOptions( \@opt_table, \@ARGV ) || exit 1;
+&Getopt::Tabular::GetOptions( \@opt_table, \@ARGV )
+    || exit $NeuroDB::ExitCodes::GETOPT_FAILURE;
 
 ################################################################
 ################ Get config setting#############################
 ################################################################
+if (!$profile ) {
```
Comment (MounaSafiHarab) [inline tools/ProdToConfig.pl]: same comment as above; looks like you added the check separately here...
but 3 lines later, you still have the condition as ` if ( $profile && !@Settings::db ) {` 
whereras I believe in other scripts, this was just `if ( !@Settings::db ) {`

**Comment by cmadjar** [comment]
@MounaSafiHarab Thanks! I corrected both of them.

**Comment by MounaSafiHarab** [comment]
merging as the code is fine and these are tools that were generated for a specific release anyway...

**Pull request merged**